Automattic/wp-calypso
# Reader Cold Start: allow user to proceed if they're following at least one blog

To "graduate" from Cold Start, we want users to follow at least one blog.

This PR enables the "OK, I'm all set!" button once the user's subscription count is 1 or more.

<img width="768" alt="screen shot 2016-06-07 at 16 53 51" src="https://cloud.githubusercontent.com/assets/17325/15864929/cd015d30-2cd0-11e6-8842-e03d5c5d711f.png">

### To test
1. In your test account, unfollow all blogs at http://calypso.localhost:3000/following/edit
2. Visit http://calypso.localhost:3000/read/start - "OK, I'm all set!" should be greyed out
3. Follow one blog from the presented cards - "OK, I'm all set!" should become active
4. Click the "OK, I'm all set!" button - you should be taken to your following stream
5. Revisit http://calypso.localhost:3000/read/start - "OK, I'm all set!" should still be active

Expected behaviour: https://cloudup.com/iU84Tr9wKrQ

Test live: https://calypso.live/?branch=add/reader/cold-start-graduation-button-for-testing
